CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # NHS.UK frontend Changelog
 
+## Unreleased
+
+:wrench: **Fixes**
+
+- `font-family` is still set (to the fallback font) when not including NHS typefaces
```

Comment:
'still' makes this read as "this does nothing" to me
'is now set to the fallback font when' reads better, perhaps?

Reply:
Good shout!